MuhanguziDavid/mydiary-frontend
# #162301966 get all entries

#### What does this PR do?
Allows all users to view entries
It creates an interface for the dashboard
Creates a navbar for protected endpoints
It also updates the project readme

#### Description of Task to be completed?
Currently, there is no UI for viewing entries, no dashboard, and no Navbar
This PR ads functionality to add a dashboard UI, retrieve entries, and show a navbar.

#### How should this be manually tested?
Run `npm install`
Run the application `npm start`
Login a user
You should see the user's entries

#### What are the relevant pivotal tracker stories?
[#162301966](https://www.pivotaltracker.com/story/show/162301966)

#### Any background context you want to add?
N/A
